samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -367,3 +389,12 @@ const struct bt_mesh_comp *model_handler_init(void)
 
 	return &comp;
 }
+
+void model_handler_start(void)
```

Comment:
Is there a reason why this is not done in `model_handler_init` like it is in the sensor server sample?

Reply:
We have both implementations. In `start` for light_ctrl and in `init` for dimmer and sensor server. Is there any logical background behind this?

Reply:
Probably not. I will keep it similar within the sensor samples by using `model_handler_init.`